### PR TITLE
fix style tag

### PR DESCRIPTION
**Pull request by letiescanciano**

### CODE REVIEW

File: modules.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 var ui = {};
-var target = document.getElementById('target');
-
+var target = document.getElementById("target");
```
Comment (letiescanciano) [inline modules.js:2]: I changed nothing about this file, it's just that my prettier change single quotes for double quotes so no need to pay attention to changes in this file.

File: public/login.js [hunk 2]
```diff
@@ -29,14 +29,7 @@ export function login() {
...
   // login
-  login.addEventListener("click", (e) => {
-    const auth = firebase.auth();
-    const promise = auth.signInWithEmailAndPassword(
-      email.value,
-      password.value
-    );
-    promise.catch((e) => console.log(e.message));
-  });
+  // login.addEventListener("click");
```
Comment (letiescanciano) [inline public/login.js:32]: At this point, `login` is undefined so this was crashing this component. It happens for all the elements that you are trying to get.
You can follow an alternative approach and define the onClick event directly on the element itself (you can check that I moved the function you had here to line 105.
This way you are sure that the element is defined.

File: public/login.js [hunk 3]
```diff
@@ -99,15 +92,32 @@ export function login() {
...
+            email.value,
+            password.value
+          );
+          promise.catch((e) => console.log(e.message));
+        }}
+      >
+        Login
+      </button>
+      <button id='signup'>SignUp</button>
+      <button id='logout' style={{ display: "none" }}>
```
Comment (letiescanciano) [inline public/login.js:117]: This was also crashing. Since it's a jsx component you need to provide inline style properties inside an object.

**Comment by coopjoslin** [comment]
Thank you @letiescanciano! I appreciate the help.

**Pull request merged**